source/NVDAObjects/window/winword.py

```diff
@@ -424,6 +451,11 @@ def activate(self):
 		mathMl=mathPres.getMathMlFromTextInfo(self)
 		if mathMl:
 			return mathPres.interactWithMathMl(mathMl)
+		newRng=self._rangeObj

```

Comment:
Yes, I meant to create a copy of range object. Changed code is in branch in_t1541_Fin
Thanks a lot for the code review.